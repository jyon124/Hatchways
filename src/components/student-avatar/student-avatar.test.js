describe("Student Avatar Component Tests", () => {
    it.todo("Renders a Student Avatar component");
});
