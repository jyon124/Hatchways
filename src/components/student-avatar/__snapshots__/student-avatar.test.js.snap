// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Avatar Component Tests Renders a Student Avatar component 1`] = `"<img src=\\"https:image.url.jpg\\" class=\\"avatar\\" alt=\\"avatar\\">"`;
